src/cmdline.rs

```diff
@@ -228,6 +237,45 @@ pub fn parse_args() -> Result<Config> {
                         .takes_value(true),
                 ),
         )
+        .subcommand(
+            SubCommand::with_name("list-stream")
+                .about("List available images in a Fedora CoreOS stream")
+                .usage("coreos-installer list-stream [OPTIONS] [<architecture> [<platform>]]")
+                .arg(
+                    Arg::with_name("stream")
+                        .short("s")
+                        .long("stream")
+                        .value_name("name")
+                        .help("Fedora CoreOS stream")
+                        .default_value("stable")
+                        .takes_value(true),
+                )
+                .arg(
+                    Arg::with_name("architecture")
+                        .value_name("architecture")
+                        .help("Target CPU architecture")
+                        .takes_value(true),
+                )
+                .arg(
+                    Arg::with_name("platform")
+                        .value_name("platform")
+                        .help("Fedora CoreOS platform name")
+                        .takes_value(true),
+                )
+                .arg(
+                    Arg::with_name("quiet")
```

Comment:
I think we want to keep this command for human usage only (as we have the original JSON for scripts), no? In that case, I don't think that we need this option.